src/sdk/namespace_s3.js

```diff
@@ -18,13 +18,14 @@ const config = require('../../config');
 class NamespaceS3 {
 
 
-    constructor({ namespace_resource_id, rpc_client, s3_params }) {
+    constructor({ namespace_resource_id, rpc_client, s3_params , active_triggers}) {
```

Comment:
Do we need `active_triggers` in the contractor? 
Can we get it from the bucket? 
We need to change all the `new NamespaceS3` with this change if we leave it here.

Reply:
I just copied this as it is in the NamespaceMerge class, there it is in the constructor too